pylint/checkers/utils.py

```diff
@@ -402,7 +402,10 @@ def is_default_argument(
     if not scope:
         scope = node.scope()
     if isinstance(scope, (nodes.FunctionDef, nodes.Lambda)):
-        for default_node in scope.args.defaults:
+        all_defaults = scope.args.defaults + [
+            d for d in scope.args.kw_defaults if d is not None
```

Comment:
We're calculating the full list of kwargs here (even if the first element of the list would return True) so we could improve performance by using a generator line 408 in ``for default_node in all_defaults:``.